tests/std/tests/P0784R7_library_support_for_more_constexpr_containers/test.cpp

```diff
@@ -583,6 +583,42 @@ static_assert(!CanDestroyN<const char*>);
 static_assert(!CanDestroyN<volatile char*>);
 static_assert(!CanDestroyN<const volatile char*>);
 
+#if defined(__clang__) && !defined(__EDG__) // TRANSITION, DevCom-10642767 (MSVC), DevCom-10896316 (EDG)

```

Comment:
We don't have EDG-imitating-Clang test configurations, so this guard is unnecessarily confusing.